### PR TITLE
update out of sync codegen

### PR DESCRIPTION
**Pull request by rakelkar**
some files appear to have gone out of sync... generated them by running make that runs hack scripts

/assign @yuzisun

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/259)

### CODE REVIEW

**Comment by jinchihe** [comment]
/hold

@rakelkar Seems that's more complex than just refresh using the script. I logged the ticket #249  to trace . Would your mind give me more time to work on this? maybe next week. Thanks.

**Comment by rakelkar** [comment]
Sure!

File: pkg/apis/serving/v1alpha1/openapi_generated.go [hunk 1]
```diff
@@ -330,7 +329,7 @@ func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenA
 				},
 			},
 			Dependencies: []string{
-				"github.com/knative/pkg/apis.Condition", "github.com/kubeflow/kfserving/pkg/apis/serving/v1alpha1.StatusConfigurationSpec"},
+				"github.com/knative/pkg/apis.Condition", "github.com/knative/pkg/apis.URL", "github.com/kubeflow/kfserving/pkg/apis/serving/v1alpha1.StatusConfigurationSpec"},
```
[the remark this answers is not in the document]
Comment (yuzisun) [reply pkg/apis/serving/v1alpha1/openapi_generated.go]: I think in knative pkg it is just an alias to `net.URL`

**Comment by jinchihe** [comment]
@rakelkar I implemented this in the #272, would you mind to close the ticket to help to review #272 ? Great thanks!

**Comment by yuzisun** [comment]
@rakelkar can you close this one since jinchi already has #272 ?

**Pull request closed without merging**